scripts/jenkins/track-upstream-internal.sh

```diff
@@ -7,7 +28,6 @@ PROJECTSOURCE=IBS/${project}
 # zypper in osc obs-service-tar_scm obs-service-github_tarballs obs-service-recompress obs-service-git_tarballs obs-service-set_version obs-service-refresh_patches
 [ -z "$PROJECTSOURCE" ] && ( echo "Error: no PROJECTSOURCE defined." ; exit 1 )
 
-export automationrepo=~/github.com/SUSE-Cloud/automation
```

Comment:
Why do you remove this line?

Reply:
I moved it to the top, as its needed above.